test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -184,6 +184,53 @@ def test_tf_serving_version_cpu(tensorflow_inference):
 
     stop_and_remove_container(container_name, ctx)
 
+# Added by hballuru to confirm tf-serving-api version is matching tf serving version
```

Comment:
Please remove the contributor name from the comment. Git blame can help one do that

Reply:
Done